_data/other.yml

```diff
@@ -1,9 +1,11 @@
 websites:
     - name: Adobe ID
       url: https://accounts.adobe.com/
-      twitter: adobecare
       img: adobe.png
-      tfa: No
+      tfa: Yes
+      sms: Yes
+      email: Yes
```

Comment:
It is there. I have also used the email option myself. 

[Manage your Adobe ID account](https://helpx.adobe.com/x-productkb/global/adobe-id-account-change.html) → [Use 2-Step Verification to strengthen account security](https://helpx.adobe.com/x-productkb/global/adobe-id-account-change.html#2-step-verification): 
> 2-Step Verification adds an extra layer of security to your account. With 2-Step Verification, while signing in, a verification code is required in addition to your Adobe ID account password. You can choose to receive this code on your mobile phone number or **on the email address associated with your account**.

[2-Step Verification: Keeping your account secure](https://helpx.adobe.com/x-productkb/global/2-step-verification-account-security.html): 
> You can receive the sign-in code on your mobile number or **on your email address**.